### DISCUSSION TITLE
Local and Public server url support

### DISCUSSION THREAD

**Discussion by mglsj** [immich-app/immich#12041] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Add support for dual server url for immich mobile app. One for the public domain and one for home network. 

When on local network and using public hostname of the server, my backup traffic routes through cloud-flare (my domain and tunnelling provider) then back to my local server. The speed is now limited by my internet connection and also eats up data cap.

I have to manually change the server address to local ip every time I need to backup, and back to public when I’m outside.

it would be helpful if the app can natively support dual server ip, trying first for local server when on wifi

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by mmomjian** [comment]
Duplicate of #308 , not planned at this time.
